packages/frontend/src/api/types.ts

```diff
@@ -93,6 +95,17 @@ export class Api {
         return permissions;
     }
 
+    /** Check if the document has been deleted. */
+    async checkDocumentIsDeleted(refId: Uuid): Promise<boolean> {
```

Comment:
Not a big deal either way, but I like to signpost that a function returns a boolean by having it start with `is`, so that this method name would shorten to `isDocumentDeleted`.

Reply:
Changed it.